libretroshare/src/ft/ftserver.cc

```diff
@@ -223,18 +223,18 @@ void ftServer::StopThreads()
 	mFtExtra->join();
 
 	delete (mFtDataplex);
```

Comment:
WTF are those?? those pointers are just nullptr as it is visible on the initializer list, those couple of lines should disappear and followinf similars too

Reply:
I'd be surprised these pointers are not set anywhere ;-) ithout them, there's no FT (especially mFtDataplex).

Reply:
They are initialized in 133:
void ftServer::SetupFtServer()
and used on all code...
I don't understand your comment @G10h4ck.

Reply:
Sorry Github diff view fooled me those deletes seemed to be inside the constructor hence my WTF reaction